conmon/conmon.c

```diff
@@ -1119,6 +1122,10 @@ int main(int argc, char *argv[])
 	        g_print("option parsing failed: %s\n", error->message);
 	        exit(1);
 	}
+	if (opt_version) {
+		puts("conmon version " VERSION "\ncommit: " GIT_COMMIT);
```

Comment:
I'm not sure why `conmon` has a glib dependency (which goes back to 530a25837), but as long as we do I think we want to stick to `g_print` here for consistency.

Reply:
I agree.